safe/gui/tools/test/test_save_scenario.py

```diff
@@ -25,7 +25,7 @@
 from safe.test.utilities import get_qgis_app, get_dock
 QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
 
-from safe.utilities.gis import qgis_version
+from safe.gui.gui_utilities import layer_from_combo
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->